hadoop-plugin/src/main/groovy/com/linkedin/gradle/hadoopdsl/BaseNamedScopeContainer.groovy

```diff
@@ -645,6 +645,30 @@ abstract class BaseNamedScopeContainer implements NamedScopeContainer {
     return scope.lookupRef(name);
   }
 
+  /**
+   * Toggle on .flow/.project files generation instead of .job/.properties files
+   *
+   * @return The value of generate_yaml_output flag - always true
+   */
+  @HadoopDslMethod
+  boolean generateYamlOutput() {
+    this.scope.bind("generate_yaml_output", true);
+    return true;
+  }
+
+  /**
+   * Creates the task to toggle between .flow/.project files generation and .job/.properties files
+   * generation.
+   *
+   * @param boolean flag Boolean to which generate_yaml_output is set
+   * @return The value of generate_yaml_output flag - always true
+   */
+  @HadoopDslMethod
+  boolean generateYamlOutput(boolean flag) {
```

Comment:
If the user passes `generateYamlOutput false` in the workflow scope, what should be the behaviour? I assume that it should not produce YAML output for consistency. Can you also add a test for this?

Reply:
Good idea - I'll add one now.